skrf/network.py

```diff
@@ -4609,7 +4675,7 @@ def reciprocity(s):
 
 ## renormalize
 def renormalize_s(s, z_old, z_new):
-    '''
+    '''wave casca
```

Comment:
typo?

Reply:
Good catch. I'll fix that on my next commit.